docs/config/index.md

```diff
@@ -415,12 +415,14 @@ export default async ({ command, mode }) => {
 
 ### server.hmr
 
-- **Type:** `boolean | { protocol?: string, host?: string, port?: number, path?: string, timeout?: number, overlay?: boolean }`
+- **Type:** `boolean | { protocol?: string, host?: string, port?: number, path?: string, timeout?: number, overlay?: boolean, runtimePort?: string }`
```

Comment:
The port is a number, so we should stick to that:

```suggestion
- **Type:** `boolean | { protocol?: string, host?: string, port?: number, path?: string, timeout?: number, overlay?: boolean, runtimePort?: number }`
```

Reply:
Because `runtimePort` is  resolved on the browser. `runtimePort` is meaningless when it's a number.